### ISSUE TITLE
Suggest better treatment for 'Quick Access' onboarding screen

### ISSUE THREAD

**Issue by rbillings** [mozilla-mobile/mozilla-vpn-client#936]
From user feedback: After login, the text "You can quickly access Mozilla VPN from your status bar." gets shown. This had me slightly confused because on WIndows, the element is called the tray icon.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/VPN-163)
┆author: Data Sync

**Comment by lesleyjanenorton** [comment]
Agreed and related: do we feel this screen is valuable? It seems more like something the user would only see once and like something that could be pulled into a future improved on-boarding flow. Right now the user will see this each and every time they sign in. @jess-cook03 what are your thoughts here?

**Comment by jess-cook03** [comment]
cc @chrisrooneys  --- backlog item for now.

**Comment by data-sync-user** [comment]
➤ Chris Rooney commented:

Jessica Cook thoughts on simply adding “system tray” to the string? So:

“You can quickly access Mozilla VPN from your status bar or system tray.”

**Comment by data-sync-user** [comment]
➤ Chris Rooney commented:

Final text to replace string in question is “You can quickly access Mozilla VPN from your status bar or system tray.”

**Comment by data-sync-user** [comment]
➤ Andrea Marchesini commented:

I think it’s better to have 2 strings: one for mac and one for linux and windows.

Chris Rooney what do you think?

**Comment by data-sync-user** [comment]
➤ Andrea Marchesini commented:

Maybe also a different string for mobile?

**Comment by data-sync-user** [comment]
➤ Andrea Marchesini commented:

Chris Rooney we need 2 new strings to fix this issue. What do you think? We can do it for 2.12?

Santiago Andrigo feedback?
